src/components/List.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import 'react-circular-progressbar/dist/styles.css';
+import Book from './Book';
+
+export default function List() {
+  return (
+    <section className="flex h-[70%] mx-16 px-4 flex-wrap gap-y-1 py-4 overflow-auto font-serif">
+      <Book title="La vida secreta de los árboles" author="A. Zambra" percentage="70" chapter="CHAPTER 17" />
+      <Book title="Conjunto Vacío" author="Verónica Gerber Bicecci" percentage="20" chapter="CHAPTER 5" />
+      <Book title="Historia de la sexualidad 1" author="M. Foucault" percentage="40" chapter="CHAPTER 6" />
+    </section>
```

Comment:
- [OPTIONAL] According to the project [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md), " `Should display a form for adding a book (no functionality yet).` ", the form for adding books should be displayed in this component, please kindly fix that.